synapse/tests/test_cortex.py

```diff
@@ -1862,6 +1862,28 @@ def v3():
 
             self.eq(core.getModlVers('grok'), 3)
 
+    def test_cortex_coremodule_telepath(self):
+        port = 0
+        link_template = 'tcp://127.0.0.1:%d/remote_core'
+        mods = (('synapse.tests.test_cortex.CoreTestModule', {'foobar': True}), )
+
+        with s_cortex.openurl('ram:///') as remote_core:
+            with s_daemon.Daemon() as dmon:
+
+                dmon.share('remote_core', remote_core)
+                link = dmon.listen(link_template % port)
+                port = link[1]['port']
+                link_url = link_template % port
```

Comment:
link options may be specified to openurl, so the string/template logic here can be replaced with
```
link = dmon.listen('tcp://127.0.0.1:0/')
port = link[1].get('port')

# ... 

with s_cortex.openurl('tcp://127.0.0.1/remote_core', port=port) as core:
    #  ....
```